pyproject.toml

```diff
@@ -30,8 +30,12 @@ dynamic = ["version"]
 repository = "https://github.com/vipyrsec/dragonfly-mainframe/"
 documentation = "https://docs.vipyrsec.dev/dragonfly-mainframe/"
 
+[build-system]
+requires = ["pdm-backend"]
+build-backend = "pdm.backend"
+
 [tool.pdm]
-version = { source = "file", path = "mypackage/__init__.py" }
+version = { source = "file", path = "src/mainframe/__init__.py" }
```

Comment:
Should this just be `mainframe/__init__.py` or is it already correct?
I'm assuming it's fine, as this is a path rather than a module?

Reply:
It's correct as is. It's relative to the project root:
> You can add a dependency with the relative paths to the project root.
> \- https://backend.pdm-project.org/metadata/#relative-paths

You can tell that it can see the file because CI is no longer failing.